### ISSUE TITLE
Narcoleptics get a weird "hourglass" shape at their cursor when they wake themselves up

### ISSUE THREAD

**Issue by Everturning** [space-wizards/space-station-14#27278]
## Description
see other areas

**Reproduction**
be narcoleptic
wake up
weird hourglass following your cursor, doesnt leave ever

**Screenshots**
![Screenshot 2024-04-23 232220](https://github.com/space-wizards/space-station-14/assets/146485322/d2931c2b-2757-49f7-97f0-ba1608413943)
**Additional context**
I am a moth wearing a french beret, which could play some part in it??? dunno.

**Comment by Everturning** [comment]
forgot to circle it, but, my cursor is being followed by that weird white thing to the left of me

**Comment by QuietlyWhisper** [comment]
that's actually a torso!

**Comment by FinnishPaladin** [comment]
a moth torso, by the looks of it.

**Comment by Everturning** [comment]
yeah i thought it was a torso too, but didnt know for sure. still real fuckin weird. on another note, it didn't seem to happen 100% of the time, so maybe it depends on how you wake up. whether you press the number or if you click it. idk.

**Comment by Everturning** [comment]
relogging WILL remove the torso, as always
someone please test this for me, i cant bcuz software

**Comment by Ian321** [comment]
Same thing just happened to me. Persisted after the round. Restarting removed it.

![Screenshot from 2024-04-27 00-04-47](https://github.com/space-wizards/space-station-14/assets/14338265/0f8a52ef-d0f7-40cf-802e-d9cd1fb78ebf)

(Cursor is on the Axe.)

**Comment by Everturning** [comment]
I cant tell if this is still around but it hasn't been closed yet so I'm bumping it
